llvm/include/llvm/CAS/ObjectStore.h

```diff
@@ -305,8 +305,8 @@ class ObjectProxy {
   }
 
   std::unique_ptr<MemoryBuffer>
-  getMemoryBuffer(StringRef Name = "",
-                  bool RequiresNullTerminator = true) const;
+  getMemoryBuffer(StringRef Name = "", bool RequiresNullTerminator = true,
+                  bool Indenpendent = false) const;
```

Comment:
Typo `Indenpendent` -> `Independent`. Same for the last thunk in `ObjectStore.cpp`.

Reply:
`Name` and `RequiresNullTerminator ` are fairly self-explanatory, but I'm not sure `Independent` is. Can you document what it does?

Reply:
That means the life time is independent from CAS. But just realize that the other function also does the same thing. Remove the unused version.